package/CHANGELOG

```diff
@@ -22,6 +22,8 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * Fixed retrieval of auxiliary information after getting the last timestep
```

Comment:
We also have a list of contributors for the upcoming version (line 20), could you add yourself there too?